Templates/FaceLocator-CSharp/run.csx

```diff
@@ -0,0 +1,90 @@
+﻿// Setup
+// 1) Go to https://www.microsoft.com/cognitive-services/en-us/computer-vision-api 
+//    Sign up for computer vision api
+// 2) Go to Function app settings -> App Service settings -> Settings -> Application settings
+//    create a new app setting FACEAPI_Subscription_Key and use Computer vision key as value
+#r "Microsoft.WindowsAzure.Storage"
+#r "Newtonsoft.Json"
+
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Newtonsoft.Json;
+using Microsoft.WindowsAzure.Storage.Table;
+using System.IO;
+
+public static async Task Run(Stream image, string name, ICollector<FaceRectangle> outTable, TraceWriter log)
```

Comment:
Change this to use an `IAsyncCollector<FaceRectangle>` and change the code to use `AddAsync`